src/defaultPane.js

```diff
@@ -21,9 +20,15 @@ module.exports = {
     return 'about '
   },
 
-  render: function (subject, dom) {
+  render: function (subject, context) {
+    var dom = context.dom
+
     var filter = function (pred, inverse) {
-      if (typeof panes.internal.predicates[pred.uri] !== 'undefined') {
+      if (
+        typeof context.session.paneRegistry.byName('internal').predicates[
+          pred.uri
+        ] !== 'undefined'
+      ) {
```

Comment:
Is this the result of `eslint --fix`? I don't want to be all bikesheddy and all, but this looks... Weird? Or is it just me?